src/Asset/utils.js

```diff
@@ -0,0 +1,13 @@
+import { SQL } from '../database'
+import { PublicationQueries } from '../Publication'
+
+export const getFindByOwnerQuery = async (owner, table) =>
```

Comment:
Even though I left another approach on `findByOwner`, the async is not necessary here

Reply:
Yeah, that was my bad 😅 I firstly made this function return `db.query` instead of just the string SQL statement, forgot to remove `async/await`. I'll remove it